src/zino/state.py

```diff
@@ -39,3 +39,16 @@ async def dump_state_to_file(filename: str = STATE_FILENAME):
     _log.debug("dumping state to %s", filename)
     with open(filename, "w") as statefile:
         statefile.write(events.json(exclude_none=True, indent=2))
+
+
+def load_state_from_file(filename: str = STATE_FILENAME):
+    """Loads and replaces Zino state from a JSON file dump"""
+    _log.info("Loading saved state from %s", filename)
+    try:
+        loaded_state = Events.parse_file(filename)
+    except FileNotFoundError:
+        _log.error("No state file found (%s), starting from scratch ", filename)
+        return
+    else:
+        global events
+        events = loaded_state
```

Comment:
If this function is unable to complete its purpose (loading state from a file), it should raise an exception (probably raise the FileNotFoundError).

Reply:
Only partial agreement: The first time Zino is started, there is no state file to load. Where and how do we handle that case?

Reply:
I might suggest that Zino should always stop if it doesn't find the state file, but that you can force-start it the very first time it runs, so that it at least generates an empty state file.

The question is whether robustness requirements dictates that it is more important that Zino is running than that it resumes its old state.  Perhaps a requirements discussion with Håvard is in order.

Reply:
If the file not existing has to be handled in this function, then renaming it to `init_state` or something like that would be more fitting